Dockerfile

```diff
@@ -1,19 +1,12 @@
 FROM registry.access.redhat.com/ubi9/go-toolset:1.20.10 AS builder
 
-COPY go.mod go.mod
-COPY go.sum go.sum
-
-RUN go mod download
-
-COPY example/ example/
-COPY githubapp/ githubapp/
-COPY Makefile Makefile
+COPY . .
 
 RUN make build
 
 FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3
 COPY --from=builder /opt/app-root/src/ci-helper-app /
```

Comment:
Just to confirm, this `ci-helper-app` is a binary that was generated by the `RUN make build` above?

Reply:
correct